### PR TITLE
[www] handle a proper menu bar

### PR DESCRIPTION
**Pull request by fabi1cazenave**

### CODE REVIEW

File: www/content/comparatif/index.md [hunk 1]
```diff
@@ -194,6 +205,12 @@ elle est *utilisable*, mais pas *optimisée* pour ça.
 - pour les Bépoètes qui ont des douleurs ou qui souhaitent tenter l’expérience
   [1DFH][], [Bépolar][] sera une alternative intéressante et très vite apprise.
 
+[ergo‑l]:     /
+[bépo]:       /bepo
+[erglace]:    /erglace
+[lafayette]:  /lafayette/#qwerty-lafayette
+[colemak ft]: /lafayette/#colemak-french-touch
+[bépolar]:    /lafayette/#bepolar
```
Comment (Nuclear-Squid) [inline www/content/comparatif/index.md:213]: Those tags are duplicates and generates errors when building the site. I’d say we move all of the link tags at the bottom of the page, but I’ll let you be the judge.

File: www/assets/css/theme.css [hunk 2]
```diff
@@ -270,17 +263,14 @@ header nav li {
...
 
-header nav li {
-  border-left: 1px solid var(--fg-banner-border);
-}
-
-header nav li:last-child {
-  border-right: 1px solid var(--fg-banner-border);
+/* submenus */
+header nav ul ul {
+  display: none;
```
Comment (Nuclear-Squid) [inline www/assets/css/theme.css]: That’s kind of sad… The default render is awful but maybe we could fix it by rendering those child elements in a separate `div` right below the nav bar, though keeping the vertical alignment could be tricky.

One *very dirty* hack we could use would be to have *both* the current style and the « other div » style in the markup, but only render one of those depending on screen width…

**Review by Nuclear-Squid — CHANGES_REQUESTED** [review]

File: www/assets/css/theme.css (hunk 2 above)
Comment (fabi1cazenave) [reply www/assets/css/theme.css]: Right now we’re limited by the markup itself, which relies on nested lists : 
```html
<nav>
  <ol>
    <li> home </li>
    <li> section
      <ol>
        <li> subitem 1 </li>
        <li> subitem 2 </li>
      </ol>
    </li>
    <li> about </li>
  </ol>
</nav>
```

The current `position: absolute;` hack allows some kind of sub-menu in desktop mode (>768px), though it’s ugly at the moment : 
image

I suspect we could have something much more flexible with this kind of nested lists, but we’d have to make sure it’s valid and accessible : 
```html
<nav>
  <ol>
    <li> home </li>
    <li> section </li>
    <ol>
      <li> subitem 1 </li>
      <li> subitem 2 </li>
    </ol>
    <li> about </li>
  </ol>
</nav>
```

And more practically, I think the general solution is to have a vertical menu in most cases : 
- in wide mode (>1200px ?) this menu could be in the left margin
- in narrow mode (<768px ?) this should be a hamburger menu

Long story short, I’m not pleased with the current solution either. It’s not *good* yet, but I think it’s *better* than what we have and a sane basis to explore from.

File: www/content/comparatif/index.md (hunk 1 above)
Comment (fabi1cazenave) [reply www/content/comparatif/index.md:213]: Indeed, this creates warnings. Let’s drop the dupes then.
